.changeset/wise-cameras-agree.md

```diff
@@ -0,0 +1,22 @@
+---
+'@astrojs/cloudflare': major
+---
+
+The configuration `build.split` and `build.excludeMiddleware` are deprecated.
+
+Configuration that were inside the astro configuration, are now moved inside the adapter:
```

Comment:
```suggestion
You can now configure this behaviour using `functionPerRoute` in your Cloudflare integration config:
```
Since this is a changeset just for Cloudflare, it's OK to talk about this in terms of the specific Cloudflare integration. I think the the key thing to hit is that this new config option is how you would replicate something you might have previously been using, so that's why I framed this as "how to get what you had". But, edit the wording to taste!